security/azure/sql_publicly_accessible_managed_instance/README.md

```diff
@@ -1,62 +1,46 @@
-- The following RightScale Credentials
-  - `AZURE_APPLICATION_ID`
-  - `AZURE_APPLICATION_KEY`
-  
-### To enable delete action
-
-Perform below steps to enable delete action.
+When a publicly accessible Azure SQL Managed Instance is detected, an email action is triggered automatically to notify the specified users of the incident. Users then have multiple actions that they are able to take after approval:
 
-- Edit the file [Check_for_publicly_accessible_Azure_SQL_Managed_Instance](https://github.com/rightscale/policy_templates/tree/master/security/azure/sql_publicly_accessible_managed_instance)
-- uncomment the line which conatins 'escalate $esc_delete_Managed_instances_approval' and save the changes.
-- upload the modified file and apply the policy.
+- *delete* - deletes the Azure SQL managed instance
+- *Note: by default *delete* action has been disabled, the user can follow the steps mentioned in "To enable delete action" section above to enable delete action.*
+- *disable public data endpoint* - modifies the configuration of virtual network of the particular SQL managed instance that allows public accessibility.
 
-### Required RightScale Roles
-- policy_designer
-- policy_manager
-- policy_publisher
-- credential_viewer
+## Input Parameters
 
-### Azure Required Permissions
+This policy has the following input parameters required when launching the policy.
 
-- Microsoft.Sql/managedInstances/*
+- *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created
+- *Exclusion Tag Key* - Azure SQL Managed instance tag to ignore instance that are with public data endpoint enabled. Only supply the tag key. The policy assumes that the tag value is irrelevant.
 
-### Installation
+## Prerequesites
 
-1. Follow steps to [Create an Azure Active Directory Application](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application)
-1. Grant the Azure AD Application access to the necessary subscription(s)
-1. [Retrieve the Application ID & Authentication Key](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-1. Create RightScale Credentials with values that match the Application ID (Credential name: `AZURE_APPLICATION_ID`) & Authentication Key (Credential name: `AZURE_APPLICATION_KEY`)
-1. [Retrieve your Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
+This policy requires the Azure Resource Manager Credential. When applying the policy select the appropriate credentials from the list for your tenant. If such credential doesn't exist please contact your cloud admin to create the Credential.
```

Comment:
see another readme for prerequisite example

Reply:
Earlier referred POL-151 for Prerequisites text, now changed it by referring to the policies which have been merged to master recently viz. Azure Rightsize SQL Databases and Azure Inefficient Instance Utilization using Log Analytics.